components/public-api/gitpod/v1/configuration.proto

```diff
@@ -106,6 +107,13 @@ message UpdateConfigurationRequest {
 
     // Specifies whether restricted_workspace_classes should be updated.
     optional bool update_restricted_workspace_classes = 3;
+
+    // restricted_editor_names specifies the editor names that are NOT allowed to be used in this configuration.
+    // If empty, all editors are allowed.
+    // Only updates if update_restricted_editor_names is true.
+    repeated string restricted_editor_names = 5;
+    // Specifies whether restricted_editor_names should be updated.
+    optional bool update_restricted_editor_names = 6;
```

Comment:
nit: Should these numbers be 4 and 5 instead?

Reply:
🙏  Addressed https://github.com/gitpod-io/gitpod/pull/19488/commits/9cf4b03a7832b1e39ef1232446e734be2484edfd